english/swami-vivekananda/teachings-of-swami-vivekananda/quotes.md

```diff
@@ -0,0 +1,9 @@
+Teachings of Shri Swami Vivekananda.
+
+﻿1. Education is not the amount of information that we put into your brain and runs riot there, undigested, all your life. We must have life building, man making, character making assimilation of ideas. If you have assimilated five ideas and made them your life and character, you have more education than any man who has got by heart a whole library..

```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.